nodejs/week5/src/backend/routes/mealsRoute.js

```diff
@@ -0,0 +1,165 @@
+
+const express = require("express");
+const router = express.Router();
+
+const dotenv = require("dotenv");
+const mysql = require("mysql");
+dotenv.config();
+
+const { MYSQL_URL } = process.env;
+const connection = mysql.createPool(MYSQL_URL);
+
+//Response all meals
+router.get('/', (req, res) => {
+    if (isEmpty(req.query)) {

```

Comment:
Instead of having all these if else. Maybe you could restructure it, so that you call a function that returns a string of sql. 
```suggestion
const sql = getSqlFromQuery(req.query);
connection.query(sql, (err, results, fields) => {
```
The inside of the `getSqlFromQuery` this is where the if and else if's would be. 
That way we abstract complexity